assets/css/stylesheet.css

```diff
@@ -1,17 +1,19 @@
 body{
```

Comment:
Hi Huish, do remember to make sure your game design fits different browser-sizes. The instructions menu does not overlap the background properly on my 1920 *1028 screen:  [https://imgur.com/a/krdaU]